ds_judgements_public_ui/sass/includes/_judgment_text_toolbar.scss

```diff
@@ -87,27 +86,24 @@
 
   margin: 0 auto;
 
-  @media (min-width: $grid__breakpoint-small) {
-    margin-left: auto;
-    margin-right: 0;
-  }
 
   &__option--pdf {
-    margin: 0 !important;
+    //margin: 0 !important;
```

Comment:
If this isn't needed any more, we should remove it entirely rather than leaving it commented out